app/assets/stylesheets/control.scss

```diff
@@ -0,0 +1,19 @@
+.control {
+  color: #AAAAAA;
+}
+
+.yo {
+  content: '1';
+}
+
+.something {
+  &.thats {
+    &.super {
+      &.nested {
+        .and-bad {
+          color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.